lms/djangoapps/ccx/views.py

```diff
@@ -430,6 +437,54 @@ def ccx_invite(request, course, ccx=None):
     return redirect(url)
 
 
+def get_email_params(request, course_id, ccx, auto_enroll, display_name_with_default=None):
```

Comment:
Nit: We already have a function by this name, `lms/djangoapps/instructor/enrollment.py:264:def get_email_params(` - perhaps this could be `get_ccx_email_params` to be more explicit?